src/acrcssc/azext_acrcssc/helper/_workflow_status.py

```diff
@@ -407,18 +416,24 @@ def generate_logs(cmd, client, run_id, registry_name, resource_group_name, await
                          run_id, registry_name, resource_group_name, e)
 
         if await_task_run:
-            polling_method = WorkflowLogPollingMethod(client, resource_group_name, registry_name, run_id)
-            LongRunningOperation(cmd.cli_ctx, progress_bar=IndeterminateProgressBar(cmd.cli_ctx, message=await_task_message))(polling_method)
+            polling_method = WorkflowLogPollingMethod(client,
+                                                      resource_group_name,
+                                                      registry_name,
+                                                      run_id)
+            LongRunningOperation(
```

Comment:
Question, is this method used to retrieve task run logs? If the run is not yet complete, it waits for completion before posting the log?

Reply:
Should we wait for the run to complete? If the run is not complete, should we log 'Running' or 'Queued‘?

In some cases, the run might take a long time to complete, such as when there is a large queue of pending runs. As a result, the CLI may either time out or the command may hang while polling for results.